### DISCUSSION TITLE
Custom template Slider

### DISCUSSION THREAD

**Discussion by Salento78** [microsoft-search/pnp-modern-search#3844] [Q&A]
Hello, I'm trying to add a check to see if the "link" property exists in a custom layout (slider).
If it does, I want to insert an tag (see image). However, the check I wrote doesn't seem to be working.
Could you help me with this?
Thanks.
![Immagine 2024-07-04 120711](https://github.com/microsoft-search/pnp-modern-search/assets/173054019/ed9976ee-bc16-4320-8209-83628b6c0abc)

**Comment by kasperbolarsen** [comment]
Are you sure that the mapping is correct? The managed property you have in the link slot must also be in the Selected Properties list. Try outputting the Link slot in your layout perhaps?
